doc/arkode/guide/source/Usage/General.rst

```diff
@@ -51,11 +51,13 @@ to the SUNDIALS core header file.
 
 .. code:: c
 
-  #include <sundials/sundials_core.h> // Provides core SUNDIALS types
-  #include <arkode/arkode_erkstep.h>  // ERKStep provides explicit RK methods.
-  #include <arkode/arkode_arkstep.h>  // ARKStep provides explicit, implicit, IMEX additive RK methods.
-  #include <arkode/arkode_mristep.h>  // MRIStep provides mutlirate RK methods.
-  #include <arkode/arkode_sprkstep.h> // SPRKStep provides symplectic partition RK methods.
+  #include <sundials/sundials_core.h>      // Provides core SUNDIALS types
+  #include <arkode/arkode_arkstep.h>       // ARKStep provides explicit, implicit, IMEX additive RK methods.
+  #include <arkode/arkode_erkstep.h>       // ERKStep provides explicit RK methods.
+  #include <arkode/arkode_sprkstep.h>      // SPRKStep provides symplectic partitioned RK methods.
+  #include <arkode/arkode_splittingstep.h> // SplittingStep provides operator splitting methods.
+  #include <arkode/arkode_forcingstep.h>   // ForcingStep provides a forcing method.
+  #include <arkode/arkode_mristep.h>       // MRIStep provides multirate RK methods.
```

Comment:
Is there some logic behind moving MRIStep to the end of this list?  In the absence of some real reason for doing this, I think that this should retain the chronological ordering, with the new steppers put at the end of the list.